Windows nightlies error (again)

On build.tox.chat:
```
In file included from src/platform/camera/directshow.cpp:25:0:
/usr/share/mingw-w64/include/amvideo.h:1518:5: error: ‘REFERENCE_TIME’ does not name a type
     REFERENCE_TIME AvgTimePerFrame;
     ^
/usr/share/mingw-w64/include/amvideo.h:1526:5: error: ‘REFERENCE_TIME’ does not name a type
     REFERENCE_TIME AvgTimePerFrame;
     ^
/usr/share/mingw-w64/include/amvideo.h:1547:5: error: ‘REFERENCE_TIME’ does not name a type
     REFERENCE_TIME AvgTimePerFrame;
     ^
In file included from src/platform/camera/directshow.cpp:28:0:
/usr/share/mingw-w64/include/dvdmedia.h:31:5: error: ‘REFERENCE_TIME’ does not name a type
     REFERENCE_TIME AvgTimePerFrame;
     ^
```

--- a/src/net/autoupdate.cpp
+++ b/src/net/autoupdate.cpp
@@ -35,8 +35,12 @@
 #include <iostream>
 
 #ifdef Q_OS_WIN
-#include <shellapi.h>
+// Because of replacing to incorrect order, which leads to building failing,
+// this region is ignored for clang-format
+// clang-format off
 #include <windows.h>
+#include <shellapi.h>
+// clang-format on
 #endif
 
 /**

--- a/src/platform/camera/directshow.cpp
+++ b/src/platform/camera/directshow.cpp
@@ -21,14 +21,18 @@
 
 #include "directshow.h"
 
-#include <QDebug>
-#include <amvideo.h>
-#include <cassert>
+// Because of replacing to incorrect order, which leads to building failing,
+// this region is ignored for clang-format
+// clang-format off
 #include <cstdint>
-#include <dvdmedia.h>
 #include <objbase.h>
 #include <strmif.h>
+#include <amvideo.h>
+#include <dvdmedia.h>
 #include <uuids.h>
+#include <cassert>
+#include <QDebug>
+// clang-format on
 
 /**
  * Most of this file is adapted from libavdevice's dshow.c,

